util/README_mtt_update.md

```diff
@@ -0,0 +1,151 @@
+# `mtt_update.py`—Minetest Translation Updater
+
+This Python script is intended for use with localized Minetest mods, i.e., mods that use `*.tr` and contain translatable strings of the form `S("This string can be translated")`. It extracts the strings from the mod's source code and updates the localization files accordingly. It can also be used to update the `*.tr` files in Minetest's `builtin` component.
```

Comment:
I think the character limit rule also applies to documentation. Feel free to correct me if I'm wrong.